### PR TITLE
Tried to add the MDN command.

### PR DESCRIPTION
**Pull request by gourabnagDev**
It uses this api ~ https://developer.mozilla.org/en-US/docs/MDN/Contribute/Tools/Search#Advanced_search_options

### CODE REVIEW

File: src/Chat/Plugin/Mdn.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        $message = sprintf("[ [%s](%s) ] %s", $result->title, $result->url, $result->excerpt);
+        $this->chatClient->postMessage($message);
+    }
+    private function postNoResult(Message $message): \Generator
+    {
+        yield from $this->chatClient->postMessage(
+            sprintf(':%s %s', $message->getOrigin(), 'Sorry I couldn\'t find a page concerning that topic on MDN.')
+        );
+    }
+}
```
Comment (PeeHaa) [inline src/Chat/Plugin/Mdn.php]: Missing trailing linebreak

File: src/Chat/Plugin/Mdn.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types=1); namespace Room11\Jeeves\Chat\Plugin; use Room11\Jeeves\Chat\Client\ChatClient; use Room11\Jeeves\Chat\Command\Command; use 
+Room11\Jeeves\Chat\Command\Message; class Mdn implements Plugin {
+    const COMMAND = 'mdn';
+    private $chatClient;
```
Comment (PeeHaa) [inline src/Chat/Plugin/Mdn.php]: Add linebreaks between properties as well as methods

**Comment by gourabnagDev** [comment]
Alright, fixing that :)

**Comment by PeeHaa** [comment]
Also I'm failry certina you are missing a `yield from`. Please actually run the bot and test it.

File: src/Chat/Plugin/Mdn.php [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    }
+    private function postResult(object $result): \Generator
+    {
+        $message = sprintf("[ [%s](%s) ] %s", $result->title, $result->url, $result->excerpt);
+        yield from $this->chatClient->postMessage($message);
+    }
+    private function postNoResult(Message $message): \Generator
+    {
+        yield from $this->chatClient->postMessage(
+            sprintf(':%s %s', $message->getOrigin(), 'Sorry I couldn\'t find a page concerning that topic on MDN.')
```
Comment (krakjoe) [inline src/Chat/Plugin/Mdn.php]: "Sorry, I couldn't do the thing ..."

Comment (gourabnagDev) [inline src/Chat/Plugin/Mdn.php]: Did you mean changing `"Sorry I couldn\'t find a page concerning that topic on MDN."` to `"Sorry, I couldn't do the thing ..."` ?

Comment (krakjoe) [inline src/Chat/Plugin/Mdn.php]: Just the comma ...

Comment (gourabnagDev) [inline src/Chat/Plugin/Mdn.php]: Oh, alright!

Comment (gourabnagDev) [inline src/Chat/Plugin/Mdn.php]: Fixed it :+1:

**Pull request closed without merging**